src/components/Modal/TodayOotd/TodayOotdModal.tsx

```
@@ -137,8 +168,20 @@ export function TodayOotdModal({
                     };
                   }}
                   renderItem={({ item }) => {
-                    const isSelected = item.image && ootdData?.image === item.image;
-                    const isEmpty = !item.image;
+                    const isOotdLayout =
+                      item.id === "ootd" &&
+                      ootdData?.items != null &&
+                      ootdData?.canvasSize != null &&
+                      ootdData.items.length > 0;
+                    const hasOotd = !!(ootdData?.ootdImage || (ootdData?.items && ootdData.items.length > 0));
```

ESLint

[error] Replace `ootdData?.ootdImage·||·(ootdData?.items·&&·ootdData.items.length·>·0)` with `⏎······················ootdData?.ootdImage·||⏎······················(ootdData?.items·&&·ootdData.items.length·>·0)⏎····················`

(prettier/prettier)